compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/codegen/CodeGenerator.java

```diff
@@ -1301,8 +1297,8 @@ public void visit(BLangAwaitExpr awaitExpr) {
         } else {
             valueRegIndex = this.getOperand(-1);
         }
-        genNode(awaitExpr.expr, this.env);
-        Operand futureRegIndex = awaitExpr.expr.regIndex;
+        genNode(awaitExpr.getExpression(), this.env);
```

Comment:
We don't need to access this via the getter.
The convention we followed throughout is, all the fields in the implemented nodes (i.e: BLang*) are public, and we can directly access it.

Shall we keep those as it is? Applicable for all places..

Reply:
Ack; The reason for changing them was to include the getter methods within the test coverage. But as it's **not** something crucial, will revert those changes to align with the usual conventions.

Reply:
Reverted with 4545a8d and 50ee9e5